src-frontend/app/controllers/procedure.js

```diff
@@ -44,17 +57,20 @@ export default Ember.Controller.extend({
         updateSortOrder: function(pageModels) {
             var controller = this;
 
+            pageModels = pageModels || controller.generateNewPageIndices()
```

Comment:
Didn't @Kapin say that everything fails when he adds JS linter to build checks? Either way, I'll create the issue